Lecture1/Task1/task4/task4.java

```diff
@@ -0,0 +1,32 @@
+package task4;
+        import java.util.Arrays;
+
+
+public class task4  {
+
+    public static void main(String[] args) {
+        int[] a = {341, 273, 278, 329, 445, 402, 388, 275, 243, 334, 412, 393, 299, 343, 317, 265};
+        String[] b = {"Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex", "Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate"};
+        boolean isSorted = false;
+        int buf;
+        String buf2;
```

Comment:
Basically naming in such kind of way can be considered as a bad practice as `buf2` isn't meaningful at all